pyproject.toml

```
@@ -38,6 +38,7 @@ dependencies = [
     "scipy>=1.7.3,<2",
     "six>=1.16.0,<2",
     "vtk>=9,<10",
+    "semver>=3.0.2",
```

```suggestion
    "semver>=3,<4",
```
We should block major versions on general deps

---

We should also add a pinned version to the "tests" extras